src/graphql/resolvers/getOfferResolver.js

```diff
@@ -1,16 +1,16 @@
-const GetOfferResolver = (db, args) => {
+const GetOfferResolver = async (db, args) => {
   const params = {
     TableName: "Offer",
     Key: {
-      id: args.id
+      offer_id: args.id
```

Comment:
Has this been tested with the DB? I think it might have to be `id`

Reply:
It's actually `offer_id` in the database. Bo made the `student_id` the partition key for the table so that we can easily get all offers for a given student. I think it was for that reason that he turned `id` into `offer_id` as to not get confused on the partition key, but maybe we should change it back to just id?

Reply:
Either way, I just wanted to make sure it lined up with the DB